lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -243,6 +332,72 @@ void FIRRTLModuleLowering::runOnOperation() {
   circuit.erase();
 }
 
+void FIRRTLModuleLowering::lowerMemoryDecls(const std::set<FirMemory> &mems,
+                                            Block *top,
+                                            CircuitLoweringState &state) {
+  auto b =
+      ImplicitLocOpBuilder::atBlockBegin(UnknownLoc::get(&getContext()), top);
+  std::array<StringRef, 6> schemaFields = {"depth",         "numReadPorts",
+                                           "numWritePorts", "readLatency",
+                                           "writeLatency",  "width"};
```

Comment:
Consider `SmallVector<StringRef, 6>`.

Reply:
C-style array (std::array) is more applicable here, as there is no reason to imply dynamic size.